index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <title>JavaScript Lesson1-4</title>
+</head>
+<body>
+    <script>
+      const tasks = {
+        '掃除' : '机を片付ける',
+        '買い物' : '牛乳を買う',
+        '運動' : '散歩する'
+      }
+
+      function showTasks() {
+        console.log('====================');
+        console.log('現在持っているタスク');
+        console.log('====================');
+
+        Object.keys(tasks).forEach((key, index) => {
+          console.log(`${index}:[内容]${tasks[key]}、[ジャンル]${key}`)
+        })
+      }
+      showTasks();
+
+      while(true) {
+        const task = prompt('タスクを入力してください');
+        const genre = prompt('ジャンルを入力してください');
+        if(task && genre) {
+          tasks[genre] = task;
```

Comment:
配列への追加は `push` が扱いやすいと思います。

```suggestion
          tasks.push({genre: genre, content: task})
```